crates/host/src/wasmbus/mod.rs

```diff
@@ -2506,7 +2506,30 @@ impl Host {
 
         debug!(actor_ref, count, "scale actor");
 
-        let actor = self.fetch_actor(&actor_ref).await?;
+        let host_id = host_id.to_string();
+        let annotations = annotations.unwrap_or_default().into_iter().collect();
+        spawn(async move {
+            if let Err(e) = self
+                .handle_scale_actor_task(&actor_ref, &host_id, count, annotations)
+                .await
+            {
+                error!("{e:?}");
+            }
+        });
+        Ok(SUCCESS.into())
```

Comment:
Nit: We might want to rename this const to `ACCEPTED` as when I first read it I was like "wait no, we don't know if it is successful yet"